python/py-luma.core/Portfile

```diff
@@ -24,7 +24,7 @@ long_description    luma.core is a component library providing a \
 checksums           rmd160  69f9aea882e320ff1429f59dcf04f25aa177d444 \
                     sha256  e6610de6db55b9ae81a5825961b85f9cf39e27a29940c34b1ae437e37d9a135e \
                     size    572562
-python.versions     38 39 310 311
+python.versions     38 39 310 311 312
 python.pep517       yes
```

Comment:
this is the default for subport for Python 3.7+ ; so it is unneeded here and can/should be removed (applies to other ports as well)

Reply:
https://github.com/macports/macports-ports/pull/22204#discussion_r1456749361:
> this is the default for subport for Python 3.7+ ; so it is unneeded here and can/should be removed (applies to other ports as well)

Thanks, updated.